### ISSUE TITLE
PDF2 is sufficient technique for SC 2.4.5 (Multiple ways, it should not

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#199]
WCAG's definition of web page is 
"a non-embedded resource obtained from a single URI using HTTP plus any other resources that are used in the rendering or intended to be rendered together with it by a user agent"

Under this normative definition, a PDF that is obtained from a single URL is a web page in its entirety, so it is not a "set of web pages" but rather a single web page, since it was obtained from a single URL. So PDF2 (Creating bookmarks) cannot be a sufficient technique since 2.4.5 is about a "set of web pages" ... but rather should be a best practise. Any single PDF downloaded from a URL automatically meets 2.4.5.

See discussion on the IG mailing list:
https://lists.w3.org/Archives/Public/w3c-wai-ig/2016JulSep/

**Comment by joshueoconnor** [comment]
@awkawk Making this advisory for 2.4.5 could be the way forward

**Comment by joshueoconnor** [comment]
Does the working group agree that PDF should be an advisory technique for 2.4.5?

**Comment by marcjohlic** [comment]
+1 to making it Advisory for 2.4.5

**Comment by awkawk** [comment]
Changes to make advisory: https://github.com/w3c/wcag/pull/249/files?diff=split

**Comment by awkawk** [comment]
CfC Approved: https://www.w3.org/WAI/GL/wiki/Decisions
